google/ggrc-core
# GGRC-3690 Preloading of data before the related issues tab is open and prevent reloading of tab when navigating to other tabs and back

# Issue description

Content on `Related Issues` tab loads each time when user open this tab.

# Steps to test the changes

1. Open Assessment info pane
2. Open `Related Issue` tab
3. Move on other tab
4. Go to `Related Issue` tab again

Actual result: content loads again
Expected result: tab should show previous state os related issue table

# Solution description

 - added preloading of content when Assessment info tab is opened
 - added caching content after user move on other tab
 - prevented refresh `Related Assessment` tab after create an Issue use `Raise an Issue` button

# Sanity checklist




# PR Review checklist
